docs/howtos/crp.md

```diff
@@ -375,66 +375,71 @@ if there are too many newly picked clusters and Fleet places resources on them a
 your backend may get overloaded. The exact pattern of interruption may vary, 
 depending on the set of resources you place using Fleet.
 
-To help minimize the interruption, Fleet provides rollout strategy configuration to help you
-transition between changes as smoothly as possible. Currently, Fleet supports only one rollout
-strategy, rolling update; with this strategy, Fleet will apply changes, including the addition or
-removal of picked clusters and resource refreshes, in an incremental manner with a number of phaes
-at a pace most appropriate for you. This is the default option and will apply to all
-changes you initiated.
+To help minimize the interruption, Fleet allows users to configure the rollout strategy, 
+similar to native Kubernetes deployment, to help transition between changes as smoothly as possible. 
+Currently, Fleet supports only one rollout strategy, rolling update. This strategy means changes, 
+including the addition or removal of picked clusters and resource refreshes, 
+will be applied in an incremental manner with a number of phases at a pace most appropriate for you.
+This is the default option and will apply to all changes you initiated.
 
 This rollout strategy can be configured with the following parameters:
 
 * `maxUnavailable` controls that, for the selected set of resources, how many clusters may become
 unavailable during a change. It can be set as an absolute number or a percentage. Default is 25%,
 and you should not use zero for this value.
+    
+    - The less value you set this parameter with, the less interruption you will experience during
```

Comment:
This could be wrtten as "the lower the value of maxUnavailable, you can expect to experience less interruptions during a change; however, this would lead to slower rollouts."

Reply:
GPT had very similar working to mine